wg-api-expression/charter.md

```diff
@@ -0,0 +1,82 @@
+# WG API Expression Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Enable declarative expression of Kubernetes APIs (metadata,  validation rules, etc.) to replace imperative, handwritten logic. This working group focuses on developing frameworks and tools that allow API authors to express API rules declaratively as part of the schema itself, improving development velocity, consistency, and maintainability across both native Kubernetes types and CRDs.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Declarative validation code generator (validation-gen) that generates Go validation code from declarative tags in API type definitions
+- Kube API Linter (KAL) for checking API types against Kubernetes API Conventions and best practices
```

Comment:
This is an ongoing subproject, not a workgroup deliverable?

Reply:
IIUC the declarative validation code generator is also [ongoing work](https://github.com/kubernetes/enhancements/issues/5073) right?